src/components/PageSections/Header/_Header.scss

```diff
@@ -5,6 +5,12 @@
   color: $coa-color-white;
 }
 
+.coa-Header--container {
+  max-width: $coa-top-menu-max-width;
+  margin-left: auto !important;
```

Comment:
just curious, what was forcing a different margin on this that makes us need it to be !important?